apps/fulfillment-e2e/src/integration/start-picking-picklist-in-progress.cy.ts

```diff
@@ -0,0 +1,56 @@
+import { PickingListsFragment } from '../support/page_fragments/picking-lists.fragment';
+
+const pickingListsFragment = new PickingListsFragment();
+
+describe('pick a picking list in progress', () => {
+  beforeEach(() => {
+    cy.clearIndexedDB();
+    cy.login();
+
+    cy.intercept('PATCH', '**/picking-lists/*', {
+      statusCode: 409,
+      body: {
+        errors: [
+          {
+            message: 'Picklist is already being picked by another user.',
+            status: 409,
+            code: '5310',
+          },
+        ],
+      },
+    }).as('picking-in-progress');
+
+    pickingListsFragment.getStartPickingButtons().eq(1).should('be.visible');
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(300);
```

Comment:
Why do we need this `wait`? Can we change it to something else? (Each wait makes tests less stable, that's why we should add them very carefully)

Reply:
Maybe it can be replaced with `cy.wait('@picking-in-progress')`? Cause I see that we add a `picking-in-progress` alias, but we don't use it